packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -185,10 +191,10 @@ export class RestAPIClient {
 	 * 	- URL
 	 * 	- site owner
 	 *
-	 * @returns {Promise<AllSitesResponse>} JSON array of sites.
+	 * @returns {Promise<AllDomainsResponse>} JSON array of sites.
 	 * @throws {Error} If API responded with an error.
 	 */
-	async getAllSites(): Promise< AllSitesResponse > {
+	async getAllSites(): Promise< AllDomainsResponse > {
```

Comment:
**Suggestion (non-blocking)**: Can we rename this function (and maybe it's docs) to reference that it's getting all the domains?

Domains and sites aren't technically interchangeable entities, as users could purchase domains unattached to a site, and could have multiple domains for a single site.

I 10000% agree this is the right API call to use for this validation, it's like 10x faster!! 😮  But, I think we should just be explicit about the API we're using and add a comment saying that we're validating with domains because it's significantly faster 👍

Reply:
> Can we rename this function (and maybe it's docs) to reference that it's getting all the domains?

Valid point. While thankfully we don't have scenarios such as that in our testing environment, you are right that domains are not necessarily sites.